src/actions/arcActions.ts

```diff
@@ -712,9 +712,9 @@ export function stakeProposal(daoAvatarAddress: string, proposalId: string, pred
       const stakingToken = await StandardToken.at(await votingMachineInstance.contract.stakingToken());
       const balance = await stakingToken.balanceOf(currentAccountAddress);
 
-      const amount: BigNumber = new BigNumber(web3.toWei(stake, 'ether'));
-      if (amount.lt(minimumStakingFee)) { throw new Error(`Staked less than the minimum: ${Number(web3.fromWei(minimumStakingFee, 'ether'))}!`); }
-      if (amount.gt(balance)) { throw new Error(`Staked more than than the balance: ${Number(web3.fromWei(balance, 'ether'))}!`); }
+      const amount: BigNumber = Util.toWei(stake);
+      if (amount.lt(minimumStakingFee)) { throw new Error(`Staked less than the minimum: ${Number(Util.fromWei(minimumStakingFee))}!`); }
```

Comment:
can still remove the "Number(..." Util.fromWei does that for you

Reply:
fixed: https://github.com/daostack/alchemy/pull/44/commits/4e486495a693d62270c25e74dbccc2792cd9cfa3

Reply:
`fromWei` returns a bignumber, not a number.  In some cases javascript will treat a bignumber like a number, but not in all.

In any case,  I think it is better to use `fromWei().toNumber()'.